### ISSUE TITLE
could not load Omni-SR models

### ISSUE THREAD

**Issue by hooke007** [chaiNNer-org/chaiNNer#1811]
**Information:**

- Chainner version: Alpha v0.18.7
- OS: Windows 10

**Description**

![Snipaste_2023-05-22_03-02-58](https://github.com/chaiNNer-org/chaiNNer/assets/41094733/dfa65592-6761-4192-97e1-a20e899419b4)

This should be a bug right? I see the support in your README.

**Logs**
[logs.zip](https://github.com/chaiNNer-org/chaiNNer/files/11526238/logs.zip)

**Comment by theflyingzamboni** [comment]
OmniSR support was added 2 days ago, so it's not in a release. It is in nightly, so you need to download the most recent build from here to use it: https://github.com/chaiNNer-org/chaiNNer-nightly/releases

If you're already using nightly, let us know, otherwise I'll close the issue.

**Comment by joeyballentine** [comment]
Yeah i should probably do a release asap. This is the downside to doing GitHub Flow instead of having release branches (readme updates get shown instantly rather than on release)

**Comment by hooke007** [comment]
I tried the latest nightly version and failed again.
[main.log](https://github.com/chaiNNer-org/chaiNNer/files/11526324/main.log)
[renderer.log](https://github.com/chaiNNer-org/chaiNNer/files/11526325/renderer.log)

**Comment by joeyballentine** [comment]
Huh... Where did you get the model from? That path file is using some weird kind of dtype that gets blocked by the security thing we have

**Comment by hooke007** [comment]
![image](https://github.com/chaiNNer-org/chaiNNer/assets/41094733/209b6168-295f-49ee-94ea-1ea3ce665a3f)
![image](https://github.com/chaiNNer-org/chaiNNer/assets/41094733/25c7e029-1c5e-42a4-afc6-f4c26dd51a7a)

**Comment by joeyballentine** [comment]
Yeah those are the right ones... Well, i know what to do to fix that but I'm super confused on how you even got this to happen lol

**Comment by hooke007** [comment]
Perhaps the official models had been updated? I tried x4 models and failed neither.

**Comment by RunDevelopment** [comment]
Just to share my perspective: I downloaded and used all pretrained OmniSR models today. I got the same error before this fix, and they worked correctly after it.

**Comment by hooke007** [comment]
I was tring to convert it to onnx. Neither fp16 or fp32 could not work. Has it not been supported yet?
<img width=400 src="https://github.com/chaiNNer-org/chaiNNer/assets/41094733/51acc72f-30d4-4e43-90fb-01630d62bfb8">

**Comment by RunDevelopment** [comment]
@hooke007 This is a different problem, so I opened a new issue: #1816. Let's continue this over there.
